src/quic/node_quic_socket.cc

```diff
@@ -1014,12 +1027,13 @@ void NewQuicSocket(const FunctionCallbackInfo<Value>& args) {
       env,
       args.This(),
       retry_token_expiration,
+      max_connections,
       max_connections_per_host,
       max_stateless_resets_per_host,
       options,
-      args[4]->IsTrue() ? QlogMode::kEnabled : QlogMode::kDisabled,
+      args[5]->IsTrue() ? QlogMode::kEnabled : QlogMode::kDisabled,
       session_reset_secret,
-      args[5]->IsTrue());
+      args[7]->IsTrue());
```

Comment:
```suggestion
      args[6]->IsTrue());
```

(I assume?)

Reply:
I really think it's supposed to be args[7] here.

Reply:
Ah, missed that it was wrong before – makes sense then :)

Reply:
I'm going to be cleaning up those arguments soon, btw... since the number of args keeps growing and most are numeric.